mne/io/kit/constants.py

```diff
@@ -150,6 +161,7 @@
 KIT.SYSTEM_UMD_2014_07 = 52  # UMD update to 16 bit ADC, July 4, 2014 -
 KIT.SYSTEM_UMD_2014_12 = 53  # UMD December 4, 2014 -
 KIT.SYSTEM_UMD_2019_09 = 54  # UMD September 3, 2019 -
+KIT.SYSTEM_YOKOGAWA_440 = 1001  # no idea
```

Comment:
@Kyung-minAn you can see the naming scheme we use lists the site and the month/year of installation of the system (at least I assume that's what this is -- @teonbrooks ? ). Do you have this information?

Reply:
@Kyung-minAn any thoughts on the naming here?

Reply:
@larsoner, I have no idea.
I think we would better to ask KIT for this information.

Reply:
For now I just went with `SYSTEM_YOKOGAWA`. We can always improve this later as it's just for internal use

Reply:
yep, I believe that these are assigned at time of installation or servicing

Reply:
Dear @teonbrooks & @larsoner 
Yes. I also believe that these are the information about the MEG systems which is installed in that date.
KIT installed several MEG systems including adult- and child-customized systems.
And there might be differences in the number of channels and/or sensor arrays.

I think that It's a good chance to check it in details.
I will contact with KIT and check it out.

@larsoner, You went well with 'SYSTEM_YOKOGAWA'. 
I wonder that the results would be changed or not, If you test with another system info (e.g. 'SYSTEM_UMD'?

Many thanks,
Min

Reply:
Dear all,
I checked this naming scheme with KIT and Ricoh company.
Your expectation is right. They mentioned that It is installation sites (University of Maryland / Yokogawa system) and date of calibration.
Thank you for developing and testing.